docs/crc.md

```diff
@@ -5,12 +5,17 @@
 ## Download CodeReady Containers for OKD
 
 Run a developer instance of OKD4 on your local workstation with CodeReady Containers built for OKD - >No Pull Secret Required!
+The [Getting Started Guide](https://crc.dev/crc/){: target=_blank} explains how to install and use CodeReady Containers.
 
-- [CodeReady Containers for OKD4 - Mac OS image](https://dl.fedoraproject.org/pub/alt/okd-crc/release/macos-amd64){: target=_blank}
-- [CodeReady Containers for OKD4 - Linux image](https://dl.fedoraproject.org/pub/alt/okd-crc/release/linux-amd64/){: target=_blank}
-- [CodeReady Containers for OKD4 - Windows image](https://dl.fedoraproject.org/pub/alt/okd-crc/release/windows-amd64/){: target=_blank}
```

Comment:
The OKD preset do not have an image for macos.
```
❯ crc setup
INFO Using bundle path /Users/tiger/.crc/cache/crc_okd_vfkit_4.11.0-0.okd-2022-08-20-022919_arm64.crcbundle 
INFO Checking if running as non-root              
INFO Checking if crc-admin-helper executable is cached 
INFO Caching crc-admin-helper executable          
INFO Using root access: Changing ownership of /Users/tiger/.crc/bin/crc-admin-helper-darwin 
Password:
INFO Using root access: Setting suid for /Users/tiger/.crc/bin/crc-admin-helper-darwin 
INFO Checking for obsolete admin-helper executable 
INFO Checking if running on a supported CPU architecture 
INFO Checking minimum RAM requirements            
INFO Checking if crc executable symlink exists    
INFO Creating symlink for crc executable          
INFO Checking if running emulated on a M1 CPU     
INFO Checking if vfkit is installed               
INFO Setting up virtualization with vfkit         
INFO Checking if CRC bundle is extracted in '$HOME/.crc' 
INFO Checking if /Users/tiger/.crc/cache/crc_okd_vfkit_4.11.0-0.okd-2022-08-20-022919_arm64.crcbundle exists 
INFO Getting bundle for the CRC executable        
choosing an image from manifest list docker://quay.io/crcont/okd-bundle:4.11.0-0.okd-2022-08-20-022919: no image found in image index for architecture arm64, variant "", OS darwin
```

Reply:
There's no issue from openshift preset so OKD image should be possible also.

Reply:
https://quay.io/repository/crcont/okd-bundle?tab=tags
Looks like missing linux/arm64 as well.

Reply:
That would be a CRC upstream issue, not OKD docs bug though. Please file this at https://github.com/code-ready/crc/issues/new/choose

> Looks like missing linux/arm64 as well.

Right, there is no arm64 OKD at all yet.

Reply:
Yes, OKD doesn't have any arm64 bit available :(, we are only producing the amd64 bits. @vrutkovs Can we put a `Note` at the start of this doc that only amd64 is supported for OKD preset. I will try to add the warning on crc side if someone wants to use it for arm64 arch.

Reply:
Okay, added